packages/storage-mixin/lib/backends/secure-ipc.js

```diff
@@ -117,6 +121,8 @@ if (typeof window !== 'undefined') {
 
     ipc.on('storage-mixin:find-one:result', listener);
 
+    // TODO: this returns a promise, but for some reason the result gets
```

Comment:
This probably just pre-dates ipcRenderer.call() and ipcMain.respondTo() being able to handle promises and responses and what not for you. Or whoever wrote this initially didn't think about or know about that.

I think I'll fix this separately because there's no reason for our changes in ipcRenderer and ipcMain anymore that I can tell - ipcMain.handle() does all this stuff for us.

I don't know at which stage electron added that stuff. Maybe it was always there? No idea.

But I think we can clean it all up in a separate PR. Or just leave this until we get rid of ampersand and then redo all this stuff.